Brocfile.js

```diff
@@ -89,10 +90,26 @@ function package(packagePath, vendorPath) {
   });
 }
 
+function packageAddon(packagePath, vendorPath) {
+  return renameFiles(pickFiles(vendorPath + packagePath, {
+    files: [ '**/*.js' ],
```

Comment:
`stew.rename` might be simpler: https://github.com/stefanpenner/broccoli-stew/blob/master/lib/rename.js#L4-L16